src/offset/utc.rs

```diff
@@ -74,18 +76,33 @@ impl Utc {
     /// let offset = FixedOffset::east(5 * 60 * 60).unwrap();
     /// let now_with_offset = Utc::now().with_timezone(&offset);
     /// ```
+    ///
+    /// # Panics
+    ///
+    /// Panics if the system clock is set to a time in the extremely distant past or future, such
+    /// that it is out of the range representable by `DateTime<Utc>`. It is assumed that this
+    /// crate will no longer be in use by that time.
+    ///
+    /// If panic in this situation is not acceptable, use [`SystemTime::now`] instead, like this:
```

Comment:
I am not sure this is an advice we want to give in general, or want to give this prominent.

Reply:
Ok, I've removed it.